internal/command/metrics/main.go

```diff
@@ -0,0 +1,27 @@
+package metrics
+
+import (
+	"io"
+
+	"github.com/urfave/cli/v2"
+)
+
+func NewApp(out io.Writer) *cli.App {
```

Comment:
I tried to use the `urfave` one, but it's much more painful than the monolith cobra, so this part needs a lot more polishing. My goal was a poc first.